### DISCUSSION TITLE
2010/11/02/%E4%B8%BA%E8%B1%86%E8%B1%86%E4%B9%B0%E4%BA%86%E4%B8%AA%E7%9B%B8%E6%9C%BA

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#299] [Announcements]
# 2010/11/02/%E4%B8%BA%E8%B1%86%E8%B1%86%E4%B9%B0%E4%BA%86%E4%B8%AA%E7%9B%B8%E6%9C%BA

我本来对摄影没啥兴趣。因为我的手比较抖，拍什么都不清楚。 不过和很多人一样，为了记录下一代的成长，我也在豆豆出生前买了一个单反。 我特意选了Sony的单反，因为它有机身防抖功能，不过试用过后，对它的防抖效果比较失望--还是不清楚。 贴一张我给老婆拍的大肚照。

https://blog.qizhen.xyz/2010/11/02/%E4%B8%BA%E8%B1%86%E8%B1%86%E4%B9%B0%E4%BA%86%E4%B8%AA%E7%9B%B8%E6%9C%BA

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Wei
2010年11月5日 12:30 编辑
欢迎加入成为SM一族。莫非是新出的A580？防抖效果要关了防抖对比对比才知道。配个定焦大光圈镜头准备拍baby吧。新出的SAL35F18还不错。找我来借也可以啊。

回复
